docs/en/Security.md

```diff
@@ -113,7 +113,9 @@ For example, the output of the shell command `ls -R` when authorization is enabl
 When user is determined, the list of groups is determined by a group mapping service, configured by
 'alluxio.security.group.mapping.class'. The default implementation is 'alluxio.security.group
 .provider.ShellBasedUnixGroupsMapping', which executes the 'groups' shell
-command to fetch the group memberships of a given user.
+command to fetch the group memberships of a given user. There is a caching mechanism for user group
+mapping, the mapping data will be cached for 60 seconds by default, the value configured by
```

Comment:
nit: this value can be configured by 'alluxio.security.group.mapping.cache.timeout.ms'.